src/main/java/org/prebid/server/auction/requestfactory/Ortb2RequestFactory.java

```diff
@@ -403,9 +407,9 @@ private Device enrichDevice(Device device, PrivacyContext privacyContext) {
 
         final Geo geo = ObjectUtil.getIfNotNull(device, Device::getGeo);
         final String countryInRequest = ObjectUtil.getIfNotNull(geo, Geo::getCountry);
-        final String country = ObjectUtil.getIfNotNull(privacyContext.getTcfContext().getGeoInfo(),
-                GeoInfo::getCountry);
-        final boolean shouldUpdateCountry = country != null && !Objects.equals(countryInRequest, country);
+        final String alpha3CountryCode = resolveCountryCode(privacyContext);
```

Comment:
Seems like method name should show, that inside it some type of conversion is being prepared.